packages/auth/src/Auth.ts

```diff
@@ -14,8 +15,8 @@ export class Auth {
   private auth: KeycloakInstance;
   private internalConfig: any;
 
-  constructor() {
-    const configuration = INSTANCE.getConfigByType(Auth.TYPE);
+  constructor(app: AeroGearApp) {
```

Comment:
Yes. I had that done in the first iteration but `SecurityService` need to get access to metrics.
Initially I tried to workaround this, but it may be actually valuable to have app as can give us flexibility.

Reply:
WDYT?

Reply:
right. In this case, I say let's just pass in the Metrics instance to the constructor of Security service. This way it will make it very obvious that Security service is depending on the metrics service, but not the `app`.

Reply:
Basing on my understanding it may look like this:
`
const app = new App()
const authConfig = app.getConfigByType(Auth.Type)
const auth = new Auth(authConfig)
`
I can do that but it seems too verbose to me. Considering that config is already known by user